.github/workflows/docker.yml

```diff
@@ -34,3 +67,28 @@ jobs:
           DOCKER_USER: github
           DOCKER_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
         run: make push-manifest-list
+      - name: Login to Stackable Harbor
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # tag=v3
+        with:
+          registry: oci.stackable.tech
+          username: robot$sdp+github-action-build
+          password: ${{ secrets.HARBOR_ROBOT_SDP_GITHUB_ACTION_BUILD_SECRET }}
+      # Used to retrieve manifest list digest after pushing to Harbor
+      - uses: imjasonh/setup-crane@v0.1
+      - name: Set up Cosign
+        uses: sigstore/cosign-installer@9614fae9e5c5eddabb09f90a270fcb487c9f7149 # tag=v3.3.0
+      - name: Build Harbor Manifest List
+        shell: bash
+        env:
+          DOCKER_USER: github
+          DOCKER_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+        run: |
+          make push-manifest-list REPO=oci.stackable.tech/sdp
+          MANIFEST_LIST_NAME=oci.stackable.tech/sdp/ubi8-rust-builder
+          # Retrieving digests of manifest list for signing
+          # This can be done with the output of docker push too, crane is the less complex solution at this point
+          DIGEST=$(crane digest $MANIFEST_LIST_NAME:latest)
```

Comment:
Sorry, one thing just came to my mind. Actually crane is not needed.
What's currently done: Push -> Get digest by tag -> sign
The purpose of signing by digest is that no one can change the tag between your push and the signing, so the digest needs to be obtained as a _returning result_ of the push. If we do it with crane, we might as well sign by tag, which is probably fine for now. It's deprecated though. So the solution would be to either sign by tag (for now) or to parse the digest from the "manifest push" output (probably with some ugly `awk` stunt).

Reply:
Done.

Unlike `docker push` , `docker manifest push` returns directly the digest of the manifest list. This made the change fairly easy! 

Thanks for your input, great catch!